docs/src/man/comparisons.md

```diff
@@ -2,11 +2,62 @@
+
+| Operations               | pandas                                                                                 | DataFrames.jl                              |
+|:-------------------------|:---------------------------------------------------------------------------------------|:-------------------------------------------|
+| Reduce multiple values   | `df.groupby('id')['x'].mean().reset_index()`                                           | `combine(groupby(df, :id), :x => mean)`    |
+| Add new columns          | `df.join(df.groupby('id')['x'].mean(), on='id', rsuffix='_mean')`                      | `transform(groupby(df, :id), :x => mean)`  |
+| Pick & transform columns | `df.join(df.groupby('id')['x'].mean(), on='id', rsuffix='_mean')[['id','x_mean','y']]` | `select(groupby(df, :id), :x => mean, :y)` |
+
+The table below compares more advanced commands:
+
+| Operations                | pandas                                                                       | DataFrames.jl                                                |
+|:--------------------------|:-----------------------------------------------------------------------------|:-------------------------------------------------------------|
+| Complex Function          | `df[['x']].agg(lambda v: np.mean(np.cos(v)))`                                | `combine(df, :x => (x -> mean(cos, x)) => :x_mean)`          |
+| Transform several columns | `df.agg({'x': max, 'y': min})`                                               | `combine(df, :x => maximum, :y => minimum)`                  |
+|                           | `df[['x','y']].mean()`                                                       | `combine(df, [:x, :y] .=> mean)`                             |
+|                           | `df.filter(regex=("^x")).mean()`                                             | `combine(df, r"^x" .=> mean)`                                |
+|                           | `df[['x', 'y']].agg([max, min]`                                              | `combine(df, ([:x, :y] .=> [maximum minimum])...)`           |
+| Multivariate function     | `df.assign(x_y_cor = np.corrcoef(df.x, df.y)[0,1])`                          | `transform(df, [:x, :y] => cor)`                             |
+| Row-wise                  | `df.assign(x_y_min = df.apply(lambda v: min(v.x, v.y), axis=1))`             | `transform(df, [:x, :y] => ByRow(min))`                      |
+|                           | `df.assign(x_y_argmax = df.apply(lambda v: df.columns[v.argmax()], axis=1))` | `transform(df, AsTable([:x,:y]) => ByRow(argmax))`           |
+| DataFrame as input        | `df.groupby('id').head(2)`                                                   | `combine(d -> first(d, 2), groupby(df, :id))`                |
+| DataFrame as output       | `df[['x']].agg(['max','min'])`                                               | `combine(:x => x -> (value = [minimum(x), maximum(x)]), df)` |
```

Comment:
```suggestion
| DataFrame as output       | `df[['x']].agg(['max','min'])`                                               | `combine(:x => x -> (value = [minimum(x), maximum(x)],), df)` |
```

Reply:
The same typo is in `dplyr` - can you please correct it. Also for pandas can you please set output column name to `value`?